Makefile

```diff
@@ -5,7 +5,6 @@ NAMESPACE=redhat-rhmi-operator
 PROJECT=integreatly-operator
 REG=quay.io
 SHELL=/bin/bash
-PREVIOUS_TAG ?=2.0.0
```

Comment:
You've removed this but not updated the gen/csv target that uses it https://github.com/integr8ly/integreatly-operator/blob/master/Makefile#L279-L280

Can you have that make target call the new script? or remove it.

Reply:
Also need to make sure this is updated https://github.com/integr8ly/integreatly-operator/blob/master/README.md#release